operations/availabilityadjustments.md

```diff
@@ -42,55 +43,60 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
+      "FirstTimeUnitStartUtc": "2023-02-19T23:00:00Z",
+      "LastTimeUnitStartUtc": "2023-02-23T23:00:00Z",
+      "UnitCount": 5,
+      "ActivityState": "Deleted",
+      "UpdatedUtc": "2023-02-28T23:00:00Z"
+    }
+  ],
+  "Cursor": "7413724a-6c48-46d4-ab3a-afae01280999"
 }
 ```
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `AvailabilityAdjustments` | array of [Availability adjustment](#availability-adjustment) | required | Availability adjustments. |
+| `AvailabilityAdjustments` | array of [Availability adjustment](availabilityadjustments.md#availability-adjustment) | required | Availability adjustments. |
 | `Cursor` | string | optional | Unique identifier of the item one newer in time order than the items to be returned. If Cursor is not specified, i.e. null, then the latest or most recent items will be returned. |
 
 #### Availability adjustment
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `Id` | string | required | Unique identifier of the [Availability adjustment](#availability-adjustment). |
+| `Id` | string | required | Unique identifier of the [Availability adjustment](availabilityadjustments.md#availability-adjustment). |
 | `AvailabilityBlockId` | string | optional | Unique identifier of the [Availability block](availabilityblocks.md#availability-block) which the availability adjustment belongs to. |
 | `ResourceCategoryId` | string | required | Unique identifier of the [Resource category](resources.md#resource-category) whose availability is updated. |
-| `FirstTimeUnitStartUtc` | string | required | Start of the time interval, expressed as the timestamp for the start of the first time unit, in UTC timezone ISO 8601 format. See [Time units](../concepts/time-units.md). |
-| `LastTimeUnitStartUtc` | string | required | End of the time interval, expressed as the timestamp for the start of the last time unit, in UTC timezone ISO 8601 format. See [Time units](../concepts/time-units.md). |
-| `UnitCount` | int | required | Adjustment value applied on the interval. |
-| `ActivityState` | string [Activity state](_objects.md#activity-state) | required | Shows whether the availability adjustment is active or deleted. |
-| `UpdatedUtc` | string | required | Last update date and time of the adjustment in UTC timezone in ISO 8601 format. |
\ No newline at end of file
+| `FirstTimeUnitStartUtc` | string | required | Start of the time interval, expressed as the timestamp for the start of the first [time unit](services.md#time-unit), in UTC timezone ISO 8601 format. |
+| `LastTimeUnitStartUtc` | string | required | End of the time interval, expressed as the timestamp for the start of the last [time unit](services.md#time-unit), in UTC timezone ISO 8601 format. |
+| `UnitCount` | integer | required | Adjustment value applied on the interval. |
+| `ActivityState` | [Activity State](_objects.md#activity-state) | required |  |
+| `ReleaseOverrideUtc` | string | optional |  |
```

Comment:
issue: This one is missing documentation.

Reply:
added in [this commit](https://github.com/MewsSystems/gitbook-connector-api/pull/620/commits/eb267b0c9317ca5b5dadebce4f7fbd373894c5ab)